jobs/aws/eks-distro/main-1-19-postsubmits.yaml

```diff
@@ -40,9 +39,7 @@ postsubmits:
         - bash
         - -c
         - >
-          make release -C projects/kubernetes/kubernetes DEVELOPMENT=false RELEASE_BRANCH=1-19 IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)'
-          &&
-          mv ./projects/kubernetes/kubernetes/_output/1-19/* /logs/artifacts
+          make postsubmit-conformance DEVELOPMENT=false RELEASE_BRANCH=1-19 IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)' ARTIFACT_BUCKET=artifactsstack-3794122512-artifactsbucket2aac5544-1f3dgu9wrpiz2
```

Comment:
Do we need to have this named that way? If our postsubmit is anyways going to run conformance, then I think we can keep the name as postsubmit

Reply:
For the ARTIFACT_BUCKET, do we need to hard code it here? Also, I thought this postsubmit was not going to call the artifacts account?

Reply:
Will remove the bucket name in the next iteration and hard code the name in the make file.